<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>anaSayfa</class>
 <widget class="QWidget" name="anaSayfa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>anaSayfa</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_C">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_F">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_K">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>K</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Hesapla</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_C">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>... C</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_F">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>... F</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_K">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>... K</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
